scout/server/blueprints/institutes/controllers.py

```diff
@@ -300,36 +299,68 @@ def populate_institute_form(form, institute_obj):
     return default_phenotypes
 
 
-def update_institute_settings(store, institute_obj, form):
-    """Update institute settings with data collected from institute form
+def get_sanger_recipients(form: MultiDict) -> List[str]:
+    """
+    Return list of Sanger recipients from form multiselect.
+    """
+    sanger_recipients = []
+    for email in form.getlist("sanger_emails"):
+        sanger_recipients.append(email.strip())
 
-    Args:
-        score(adapter.MongoAdapter)
-        institute_id(str)
-        form(dict)
+    return sanger_recipients
 
-    Returns:
-        updated_institute(dict)
 
+def get_clinvar_submitters(form: MultiDict) -> Optional[List[str]]:
+    """
+    Return list of ClinVar sumbitters from form multiselect.
+    This is not available on the form for unprivileged users, only admin.
+    """
+
+    clinvar_submitters = None
+    if current_user.is_admin:
+        clinvar_submitters = []
+        for email in form.getlist("clinvar_emails"):
+            clinvar_submitters.append(email.strip())
+    return clinvar_submitters
+
+
+def get_loqusdb_ids(form: MultiDict) -> Optional[List[str]]:
+    """
+    Return loqusdb ids from the form multiselect.
+    This is not available on the form for unprivileged users, only admin.
+    """
+    if current_user.is_admin is False:
+        return None
+
+    return form.getlist("loqusdb_id")
+
+
+def get_gene_panels(store: MongoAdapter, form: MultiDict, tag: str) -> Dict:
```

Comment:
This works, but you could also remove the string tag here with a boolean panels_matching=False, then if the panels_matching=True you could use get gene_panels_matching from the form instead of gene_panels? I don't know, I prefer working with booleans when possible